src/components/dashboard/IconSection.scss

```diff
@@ -0,0 +1,50 @@
+@import "~styles/partials/mixins";
+@import "~styles/partials/colors";
+
+.root {
+
+  display: flex;
+  flex-direction: row;
+  position: relative;
+
+  .col {
+    flex: 1 1 50%;
+    position: relative;
+  }
+
+  .left {
+    @include position(absolute);
+    display: flex;
+    flex-direction: column;
```

Comment:
as workaround you must wrap it in `display: flex; flex-direction: row`. In other words <div> should'n be first parent with 'column'